Backpack/Button/Classes/BPKButton.m

```diff
@@ -728,150 +591,118 @@ - (void)updateLoadingState:(BOOL)loading {
     if (destructiveContentColor != _destructiveContentColor) {
         _destructiveContentColor = destructiveContentColor;
-        [self updateContentColor];
+        [self updateAppearance];
+        [self updateTitle];
     }
 }
 
 - (void)setDestructiveBackgroundColor:(UIColor *_Nullable)destructiveBackgroundColor {
     if (destructiveBackgroundColor != _destructiveBackgroundColor) {
         _destructiveBackgroundColor = destructiveBackgroundColor;
-        [self updateBackgroundAndStyle];
+        [self updateAppearance];
     }
 }
 
 - (void)setDestructiveBorderColor:(UIColor *_Nullable)destructiveBorderColor {
     if (destructiveBorderColor != _destructiveBorderColor) {
         _destructiveBorderColor = destructiveBorderColor;
-        [self updateBackgroundAndStyle];
+        [self updateAppearance];
     }
 }
 
 - (void)setLinkContentColor:(UIColor *_Nullable)linkContentColor {
     if (linkContentColor != _linkContentColor) {
         _linkContentColor = linkContentColor;
-        [self updateContentColor];
+        [self updateAppearance];
+        [self updateTitle];
     }
 }
 
-+ (UIColor *)highlightedWhite {
-    return [BPKColor blend:BPKColor.white with:BPKColor.skyGray weight:0.85f];
-}
-
-+ (UIColor *)highlightedOutline {
-    return [BPKColor.white colorWithAlphaComponent:0.8];
-}
-
-+ (UIColor *)highlightedBlue {
-    return [BPKColor blend:BPKColor.primaryColor with:BPKColor.skyGray weight:0.85f];
-}
-
-+ (UIColor *)highlightedRed {
-    return [BPKColor blend:BPKColor.systemRed with:BPKColor.skyGray weight:0.85f];
-}
-
 // Note this is needed as the system does not correctly respond to the trait collection change to update the background
 // color.
 - (void)traitCollectionDidChange:(nullable UITraitCollection *)previousTraitCollection {
     [super traitCollectionDidChange:previousTraitCollection];
 #if __BPK_DARK_MODE_SUPPORTED
     if (self.traitCollection.userInterfaceStyle != previousTraitCollection.userInterfaceStyle) {
-        [self updateBackgroundAndStyle];
-        [self updateContentColor];
+        [self updateAppearance];
+        [self updateTitle];
     }
 #endif
 }
 
-+ (UIColor *)disabledBackgroundColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint06 darkVariant:BPKColor.blackTint02];
-}
-
-+ (UIColor *)disabledContentColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint04 darkVariant:BPKColor.blackTint05];
-}
-
-+ (UIColor *)disabledBorderColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint03 darkVariant:BPKColor.blackTint05];
-}
-
-+ (UIColor *)boxyBackgroundColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.white darkVariant:BPKColor.blackTint02];
-}
-
-+ (UIColor *)boxyBorderColor {
-    return [BPKColor dynamicColorWithLightVariant:BPKColor.skyGrayTint06 darkVariant:BPKColor.skyGrayTint02];
-}
-
 + (CGFloat)buttonTitleIconSpacing {
+    // TODO change to BPKSpacingMd
```

Comment:
This is here for future further alignment, but don't want to change sizes and colours in the same PR